.buildconfig.yml

```diff
@@ -72,6 +72,10 @@ projects:
     path: components/feature/media
     description: 'Feature component for website media related features.'
     publish: true
+  feature-loginsexceptions:
```

Comment:
Does it make sense to turn this into `feature-logins` so that we could land other things related to logins here too in the future?